src/conmon.c

```diff
@@ -277,6 +278,50 @@ static log_level_t parse_level(char *level_name) {
   nexitf("No such log level %s", level_name);
 }
 
+
+/* Different log drivers */
+typedef enum {
+  KUBERNETES_LOG_FILE,
+  JOURNALD,
+  INVALID,
+} log_driver_t;
+
+// global saving the log driver the user inputted
+static log_driver_t g_log_driver = INVALID;
+
+// global saving the log path to use if the log driver is kubernetes-log-file
+static char* g_log_path = NULL;
+
+// Value the user must input for each log driver
```

Comment:
If I understand the patch, it will drop the "json-file" in favor of "kubernetes-log-file"?